src/app/components/item-hover.tsx

```diff
@@ -30,9 +31,10 @@ type HoverBind = ReturnType<typeof useHover>[1];
 type FocusBind = ReturnType<typeof useFocus>[1];
 
 interface DefaultSpreadProps extends HoverBind, FocusBind {
+  bg: 'transparent';
```

Comment:
What's the point of a string literal here? If it can only be one value, it doesn't need to be a prop

Reply:
I had to set this now or it was white and hid the content (I think that was it). I can check it again, but removing @stacks/ui here did break it.

Reply:
Lets collect stuff like this in a list and we can tackle that in future after we've done the big job of removing the library